src/terminal/scroll/init.lua

```diff
@@ -0,0 +1,88 @@
+--- Terminal Scroll Module
+-- Provides utilities for handling scroll regions in terminals.
+-- @module scroll
+local M = {}
+local output = require("terminal.output")
+
+-- Register the module early to avoid circular dependencies
+package.loaded["terminal.scroll"] = M
+
+--- Function to return the default scroll reset sequence
+-- @treturn string The ANSI sequence for resetting the scroll region.
+function M.scroll_reset()
```

Comment:
this function should have the `s` postfix, since it returns a string, to be following the same principle as the other ones.

That said; we should then also add the function without the `s` that applies the reset by writing it to the output steam, for it to be the same across the board.